client/src/styles/index.scss

```diff
@@ -157,8 +157,10 @@ main[page="details"] a.navbar-brand {
 
 main[page="details"] span.food_name {
   position: absolute;
-  bottom: 8em;
-  left: 0em;
+  bottom: 10em;
```

Comment:
### Nitpick

I'll advise you stick to one unit. If you're going to use `em` (which I advise you don't because of responsiveness issue - `rem` is a better choice) or `px` ensure all units in your codebase are converted to follow that unit.